Application/EdFi.Ods.Common/Configuration/ApiSettings.cs

```diff
@@ -6,14 +6,18 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text.RegularExpressions;
 using EdFi.Common.Configuration;
 using EdFi.Common.Extensions;
+using EdFi.Ods.Common.Extensions;
```

Comment:
This reference is not being used, can be removed

Reply:
Done.